ArjunaJTA/jms/src/main/java/org/jboss/narayana/jta/jms/ConnectionFactoryProxy.java

```diff
@@ -39,11 +39,21 @@
 
     private final TransactionHelper transactionHelper;
 
+    /**
+     * @param xaConnectionFactory factory to get XA connection instances.
```

Comment:
Do you need a NotNull on the xaConnectionFactory - it is used elsewhere without any null checks